lib/utilities/strategy-helpers/smi-canary-deployment-helper.js

```diff
@@ -8,10 +8,12 @@ const TaskInputParameters = require("../../input-parameters");
 const fileHelper = require("../files-helper");
 const helper = require("../resource-object-utility");
 const utils = require("../manifest-utilities");
+const kubectlUtils = require("../../kubectl-util");
 const canaryDeploymentHelper = require("./canary-deployment-helper");
 const utility_1 = require("../utility");
 const TRAFFIC_SPLIT_OBJECT_NAME_SUFFIX = '-workflow-rollout';
 const TRAFFIC_SPLIT_OBJECT = 'TrafficSplit';
+var trafficSplitAPIVersion = null;

```

Comment:
Use `let` instead

Reply:
Set it to empty string or a set a default value instead of null

Reply:
Okay